modules/retail/pages/retail-mass-config.adoc

```diff
@@ -1,85 +1,90 @@
 [[retail.mass.config]]
 = Mass Configuration
 
+Mass configuration is possible with branch servers and terminals.
+
+
+
 [[retail.branch_mass_config]]
-== Branch server mass configuration
+== Branch Server Mass Configuration
 
 Branch servers are configured individually using formulas.
-If you are working in an environment with many branch servers, you might find it easier to configure the terminals automatically with a pre-defined configuration file, rather than configuring each branch server individually.
+If you are working in an environment with many branch servers, you might find it easier to configure branch servers automatically with a pre-defined configuration file, rather than configuring each one individually.
 
 // https://gitlab.suse.de/SLEPOS/SUMA_Retail/tree/master/python-susemanager-retail
 
 
 
-=== Configure multiple branch servers
+=== Configure Multiple Branch Servers
 
-This procedure requires the [package]``python-susemanager-retail`` package, which is provided with {smr}.
-Install the [package]``python-susemanager-retail`` package on the {susemgr} server.
+Configuring multiple branch servers requires the [package]``python-susemanager-retail`` package, which is provided with {smr}.
+Install the [package]``python-susemanager-retail`` package on the {productname} server.
 
-.Procedure: Configuring multiple branch servers
+.Procedure: Configuring Multiple Branch Servers
 
 . Create a YAML file describing the infrastructure you intend to install.
-An example YAML file is at the end of this chapter, for you to reference.
+For an example YAML file, see <<retail.mass.config.example.yaml>>.
```

Comment:
I wonder if it would be better to put this in a separate file and use an xref? That's how we've handled example files in the other docs.